src/Tribe/Tickets_Handler.php

```diff
@@ -851,6 +881,75 @@ public function do_meta_box( $post ) {
 		include $this->path . 'src/admin-views/meta-box.php';
 	}
 
+
+	/**
+	 * Render the ticket row into the ticket table
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $ticket
+	 * @param object ticekt provider object
```

Comment:
There's a typo here ('ticekt') but more importantly - I think you've taken my note about adding param tags for the `tribe_events_tickets_ticket_table_add_tbody_column` hook and applied it here (to the method) by mistake